src/main/java/randoop/generation/ComponentManager.java

```diff
@@ -213,18 +233,19 @@ void clearGeneratedSequences() {
             literals = (literals == null) ? sl : new ListOfLists<>(literals, sl);
           }
         }
+      } else if (classLiterals != null && Randomness.weightedCoinFlip(GenInputsAbstract.p_const)) {
```

Comment:
This seems to change the default behavior of Randoop.  Shouldn't it depend on `literals_level` being `CLASS_OR_ALL`?